build-tools/packages/build-cli/src/index.ts

```diff
@@ -4,4 +4,4 @@
  */
 
 export { run } from "@oclif/core";
-export type { ReleaseGroup, ReleasePackage } from "./releaseGroups";
+export type { knownReleaseGroups, ReleaseGroup, ReleasePackage } from "./releaseGroups";
```

Comment:
Does it need to be exported?

Reply:
Alas, it does. I don't remember why I export the ReleaseGroup type, but I do, so I have to export this as well. I don't think any of it needs to exported, but that's a breaking change (though practically it's probably not a big deal) so I skipped it.